src/programs/match_template/match_template.cpp

```diff
@@ -1190,6 +1162,8 @@ bool MatchTemplateApp::DoCalculation()
 		best_psi.Rotate2DInPlaceBy90Degrees(false);
 		// To account for the pre-rotation, psi needs to have 90 added to it.
 		best_psi.AddConstant(90.0f);
+		// We also want the angles to remain in (0,360] so loop over and clamp
+		for (int idx = 0; idx < best_psi.real_memory_allocated; idx++) { best_psi.real_values[idx] = clamp_angular_range(best_psi.real_values[idx], true); }
```

Comment:
This clamps to [-180,180], right? Would this work:

```suggestion
		for (int idx = 0; idx < best_psi.real_memory_allocated; idx++) { best_psi.real_values[idx] = fmodf(best_psi.real_values[idx], 360.0f); }
```

Reply:
Oh right, I only looked to see that the cases like 450 had been fixed, i didn't stop to read my own comment (-180,180) which is ironic given my last response that I prefer to name methods after their actions.

I'll rename clamp_angular_range -> clamp_angular_range_-180_to_180 add clamp_angular_range_0_to_360